tools/build_pak_files.py

```diff
@@ -42,9 +42,12 @@
 
     assert os.path.exists(shared_dir)
     unreal_project_content_shared_dir = os.path.join(unreal_project_content_dir, "Shared")
+    unreal_project_content_shared_dir_symlink_created = False
+
     if not os.path.exists(unreal_project_content_shared_dir):
         print(f"[SPEAR | build_pak_files.py] Creating symlink: {unreal_project_content_shared_dir} -> {shared_dir}")
         os.symlink(shared_dir, unreal_project_content_shared_dir)
+        unreal_project_content_shared_dir_symlink_created = True
 
     assert os.path.exists(scenes_dir)
     scene_names = [ os.path.basename(x) for x in glob.glob(os.path.join(scenes_dir, "*")) ]
```

Comment:
- let's use os.listdir here because it's simpler, and we're not really using glob for it's intended use, which is handling wildcards